### ISSUE TITLE
Can't purchase available Gems, or anything from Rewards column requiring Gold

### ISSUE THREAD

**Issue by Indubitably1** [HabitRPG/habitica#5443]
I was away from HabitRPG for awhile, and was resting in the Inn, and since I have come back this past week I haven't been able to purchase any Rewards with Gold, or purchase Gems, despite having 40 available. The Gold debits from the account but the item isn't purchased, and when I refresh the page, the Gold is back in the account. When I try to purchase Gems, the transaction seems to go through, but then when I refresh they disappear as well and the number available returns to 40.

Any idea what might be going on?

User ID: 57aef3cc-7fe3-4d34-bbb2-26260c83d616

**Comment by Alys** [comment]
Can you please try this on a different browser?

If that's impossible, please log out, clear your browser's cache, and your browser's local storage (if you're not sure how, you might be able to google for instructions for your browser), and then try again?

**Comment by Indubitably1** [comment]
Hi Alys, 

I have been using Firefox 35.0.1, so I tried an older Safari browser (6.0.5) and everything appears to be working fine. I'll use Safari for now and try the suggestions you gave for clearing the browser cache, etc. and updating Firefox, then seeing if that works.

Thanks for your help!

**Comment by Indubitably1** [comment]
I just wanted to mention that I had not made any changes to the browser from the time I went away from HabitRPG until the time I came back (over a month). After logging out and back in, clearing the browser's cache, and local storage, and updating, this issue persists. In fact, now the Gold just disappears from the window when I click on an item in the Rewards without the the usual pop-up/tag thing appearing, and the item still doesn't purchase. 

Can anyone suggest what might be going on and what I can do to get this thing working? I'm not really willing to permanently switch browsers when it works for every other website I regularly use.

**Comment by crookedneighbor** [comment]
One clarifying question: When you say the rewards, you mean the market, not the rewards column on the tasks page, right? Gems purchasable with gold can (currently) only be purchased in the market. 

If you meant the rewards column on the task page, I'm curious to know what you're clicking on.

**Comment by Indubitably1** [comment]
I mean the Rewards column on the Tasks page. And yes, the issue also occurs for Gems purchased with gold in the Market. 

What I am clicking on in the Rewards column is the available equipment, Health Potion, Enchanted Armoire, etc.

**Comment by crookedneighbor** [comment]
Sorry, misread your issue at first. I thought you were trying to buy gems from the rewards column (that's what I get for reading Github issues when I first get out of bed)

I just tried it in Firefox 38.0.5 and was not able to reproduce the issue. Did you try upgrading your browser? Or are you still using Firefox 35? Do you have any plugins or extensions enabled that might be interfering with the site?

**Comment by Indubitably1** [comment]
No worries!

I upgraded to Firefox 38.0.5 but that did not help. So I took your suggestion and checked the Firefox Add-ons, and since there are only a couple in there, I tried disabling them one at a time and then retesting the site as I disabled each one. It turns out the extension causing the issue is AdBlock Plus 2.6.9.1. It seems that "disable on habitrpg.com" had become unchecked, causing this whole debacle.

Lesson learned; thank you for your patience and assistance!
